uint32 layer_level
uint32[] index_list_relative
---
ram_msgs/AdditiveManufacturingPose[] poses
